source/index.html.haml

```diff
@@ -5,34 +5,35 @@ title: Hey, I'm Amber Aultman
-                    = link_to 'AIGA Jacksonville', 'https://jacksonville.aiga.org/', :target => "_blank"
-                    and a former teacher at 
-                    = link_to 'Bloc.io.', 'https://www.bloc.io/', :target => "_blank"
-                %p.large
-                    I’m currently accepting speaking and workshop facilitation engagements both locally and abroad. 
-                = link_to 'Get in touch', 'mailto:amber.aultman@gmail.com', :class => "button-primary"
-            .links
-                = link_to 'LinkedIn', 'https://www.linkedin.com/in/amberaultman/', :target => "_blank", :class => "button-tertiary"
-                = link_to 'Medium', 'https://medium.com/@amberaultman', :target => "_blank", :class => "button-tertiary"
-                = link_to 'Github', 'https://github.com/amberaultman', :target => "_blank", :class => "button-tertiary"
-                = link_to 'Email', 'mailto:amber.aultman@gmail.com', :class => "button-tertiary"
-            .close
-                %ion-icon{:name => "close-outline"}
+        .wrapper
+            %img{:src => "https://via.placeholder.com/600x600"}
+            %article
+                .bio
+                    %p.large
+                        I’m Amber Aultman, a design leader currently living in Phoenix, AZ. I’m the Head of Design at
+                        = link_to 'Tuft & Needle', 'https://www.tuftandneedle.com/', :target => "_blank"
+                        where I'm responsible for the visual experience of our brand: everything from product packaging and retail signage to our internal apps, e-commerce platform, and other digital channels. I’m crazy passionate about mentoring my team and making sure the customer experience stays paramount as we transistion through a merger. I’m also the past President of
+                        = link_to 'AIGA Jacksonville', 'https://jacksonville.aiga.org/', :target => "_blank"
+                        and a former teacher at
+                        = link_to 'Bloc.io.', 'https://www.bloc.io/', :target => "_blank"
+                    %p.large
+                        I’m currently accepting speaking and workshop facilitation engagements both locally and abroad.
+                    = link_to 'Get in touch', 'mailto:amber.aultman@gmail.com', :class => "button-primary"
+                .links
+                    = link_to 'LinkedIn', 'https://www.linkedin.com/in/amberaultman/', :target => "_blank", :class => "button-tertiary"
+                    = link_to 'Medium', 'https://medium.com/@amberaultman', :target => "_blank", :class => "button-tertiary"
+                    = link_to 'Github', 'https://github.com/amberaultman', :target => "_blank", :class => "button-tertiary"
+                    = link_to 'Email', 'mailto:amber.aultman@gmail.com', :class => "button-tertiary"
+                %button.close
+                    %ion-icon{:name => "close-outline", :onclick => "showHide()"}
```

Comment:
I converted your close icon to a `button` element and added the `showHide()` click handler method here, so we can trigger the open/close animation from either the top button or this bottom button.